Notify missing gnosis pay api key on manual redecode (#10931)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :bug:`10862` rotki will now show a warning when a Gnosis Pay transaction is manually redecoded but there are no Gnosis Pay api keys in the DB.

Reviewer: it is not api keys. I would say

```suggestion
* :bug:`10862` rotki will now show a warning when a Gnosis Pay transaction is manually redecoded but there are no Gnosis Pay is not connected to rotki.
```

> * :bug:`10862` rotki will now show a warning when a Gnosis Pay transaction is manually redecoded but there are no Gnosis Pay api keys in the DB.

Reviewer 2: ```suggestion
* :bug:`10862` rotki will now show a warning when a Gnosis Pay transaction is manually redecoded but Gnosis Pay is not connected.
```
